engine/apps/schedules/tasks/notify_about_gaps_in_schedule.py

```diff
@@ -79,20 +80,21 @@ def notify_about_gaps_in_schedule(schedule_pk):
         task_logger.info(f"Tried to notify_about_gaps_in_schedule for non-existing schedule {schedule_pk}")
         return
 
-    today = timezone.now().date()
-    gaps = list_of_gaps_in_schedule(schedule, today, today + timezone.timedelta(days=7))
-    schedule.gaps_report_sent_at = today
+    now = timezone.now()
+    events = schedule.final_events(now, now + datetime.timedelta(days=7), with_empty=False)
```

Comment:
You are right, we can filter it in the task. It makes sense since we need to filer the output anyway.

I think, if we change using of `check_gaps_for_next_week` we should also change using `check_empty_shifts_for_next_week` since they have pretty similar logic. I would propose to do it in separate PR